### ISSUE TITLE
Exception: current block is None

### ISSUE THREAD

**Issue by JulianOestreich90** [metebalci/pdftitle#40]
For some of my academic papers exception occur:
`Exception: current block is None, this might be a bug. please report it together with the pdf file`

The papers are:
- "Why Does Social Exclusion Hurt? The Relationship Between Social and Physical Pain" (MacDonald & Leary 2005)
- "Psychology and Epigenetics" (Masterpasqua 2009)

**Comment by metebalci** [comment]
Thanks for reporting, I will check but not sure how quick I can figure the problem out.

**Comment by metebalci** [comment]
There are two different issues here:

For Masterpasqua 2009, there are lines/dashes (-) in the first page that have actually the largest font size, and the actual title has the second largest font size. This means the title can be obtained by using eliot algorithm with eliot-tfs parameter 1 (meaning the second largest font):

```
$ pdftitle -a eliot --eliot-tfs 1 -p 2009_Masterpasqua_Psych\&Epi_Rev\ Gen\ Psych.pdf
Psychology and Epigenetics
```

For MacDonald & Leary 2005, the issue is complicated and related to XForms which is not supported yet. It is also reported as #39. I am closing this issue and follow the XForms issue from that one.

**Comment by metebalci** [comment]
The problem with XForms are fixed with 0.15 but there is another problem and as an example it affects this particular pdf, MacDonald & Leary 2005. Experimenting with `--translation-heuristic` option for this issue. Documented under #43
